website_sale_suggest_create_account/hooks.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2020 Alexandre Díaz - Tecnativa S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import SUPERUSER_ID, api
+from odoo.tools import config
+
+
+def post_init_hook(cr, registry):
+    if config["test_enable"] or config["test_file"]:
+        with api.Environment.manage():
+            env = api.Environment(cr, SUPERUSER_ID, {})
+            env.ref("website_sale_suggest_create_account.cart").active = False
+            env.ref(
+                "website_sale_suggest_create_account.short_cart_summary"
+            ).active = False
```

Comment:
Why do you need this???

Reply:
Explained here: https://github.com/OCA/e-commerce/pull/336/files#diff-981e540460b2382c62143803bf7ae257R8